src/glog/logging.h.in

```diff
@@ -647,6 +647,10 @@ void MakeCheckOpValueString(std::ostream* os, const signed char& v);
 template <> GOOGLE_GLOG_DLL_DECL
 void MakeCheckOpValueString(std::ostream* os, const unsigned char& v);
 
+// Provide printable value for nullptr_t
+template <> GOOGLE_GLOG_DLL_DECL
+void MakeCheckOpValueString(std::ostream* os, const std::nullptr_t& v);
```

Comment:
This would not compile under C++98/03, would it? This needs a guard. Same in the `.cc`.